core/network/types/collator_messages.hpp

```diff
@@ -28,123 +33,109 @@ namespace kagome::network {
    * CollatorStatement)
    */
 
+  using Dummy = std::tuple<>;
+
   /**
    * Collator -> Validator message.
-   * Declaration of the intent to advertise a collation.
+   * Advertisement of a collation.
    */
-  struct CollatorDeclaration {
-    /*
-     * Public key of the collator.
-     */
-    consensus::grandpa::Id collator_pubkey;
-
-    /*
-     * Parachain Id.
-     */
-    uint32_t para_id;
-
-    /*
-     * Signature of the collator using the PeerId of the collators node.
-     */
-    consensus::grandpa::Signature collator_signature;
-  };
+  using CollatorAdvertisement =
+      std::tuple<primitives::BlockHash  /// Hash of the parachain block.
+                 >;
```

Comment:
Should this wait for #1309?

Reply:
[branch tuple/collation](https://github.com/soramitsu/kagome/blob/tuple/collation/core/network/types/collator_messages.hpp)?